Assets/Mirror/Runtime/NetworkClient.cs

```diff
@@ -27,7 +27,8 @@ public class NetworkClient
 
         internal static ConnectState connectState = ConnectState.None;
 
-        public static string serverIp { get; internal set; } = "";
+        [Obsolete("Use NetworkClient.connection.address instead.")]
+        public static string serverIp => connection.address;
```

Comment:
can we do this without obsolete? having NetworkClient.address might still be valuable for ease of use.
but it's definitely good to not have extra state and just point to connection.address

Reply:
Sure.